config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  resources :grantees
```

Comment:
[suggestion] only define the route we are currently using, so
`resources :grantees, only: %i[show]`